### PR TITLE
Little path things

### PR DESCRIPTION
**Pull request by lgierth**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1054,7 +1046,7 @@ func tabPrintSortedMap(headers []string, m map[string]string) {
 func getGoPath() (string, error) {
 	gp := os.Getenv("GOPATH")
 	if gp == "" {
-		return "", fmt.Errorf("GOPATH not set")
+		return homedir.Expand("~/go")
```
Comment (whyrusleeping) [inline main.go]: I don't think this is safe to assume. For my setup this points to my go installation (aka GOROOT) and not my GOPATH

Comment (lgierth) [inline main.go]: Hence you probably set GOPATH to something else :) This change just mirrors the default assumed by the go tool.

Comment (whyrusleeping) [inline main.go]: Oh? I didnt know go assumed that. Neato.

Comment (lgierth) [inline main.go]: Yeah since 1.8 :motor_boat:

**Pull request merged**